src/main/resources/public/js/components/leftPane.js

```diff
@@ -0,0 +1,168 @@
+let leftPane = app.component('Leftpane', {
+    emits: ['update-err', 'cancel-rec-rename'],
+    data () {
+        return {
+            showAddRec: false,
+            recName: ''
+        };
+    },
+    mounted () {
+        $.get('/recipes', 'json')
+            .done(
+                (recipeList) => {
+                    this.$store.commit('updateRecList', recipeList);
+                    this.$store.commit('sortRecList');
+                },
+            ).fail(this.failureOnAjaxOfRecipe);
+    },
+    methods: {
+        showAddRecDiv () {
+            this.showAddRec = true;
+        },
+        clickRec (idx) {
+            this.$store.commit('activateRec', idx);
+            this.$store.commit('setCurRec', this.$store.state.recipeList[idx]);
+            this.$emit('cancel-rec-rename');
+        },
+        addRec () {
+            const rec = {
+                name: this.recName,
+                ingredients: [
+                ],
+                id: this.$store.state.recipeList.length,
+                active: false,
+            };
+            $.post({
+                url: 'recipes',
+                data: JSON.stringify(rec),
+                contentType: 'application/json',
+                dataType: 'json',
+            }).done(
+                (recipe) => {
+                    // document.getElementById('success').play();
```

Comment:
Consider uncommenting this and verifying it works. Same for other places where you commented out the audio.